lib/ansible/plugins/lookup/lmdb_kv.py

```diff
@@ -0,0 +1,104 @@
+# (c) 2017-2018, Jan-Piet Mens <jpmens(at)gmail.com>
+# (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: lmdb
+    author:
+      - Jan-Piet Mens (@jpmens) <jpmens(at)gmail.com>
+      - Ansible Core
+    version_added: "2.5"
```

Comment:
@gundalow this would be a good PR for the community review session - if there's interest, the `version_added` would need to be updated.